### ISSUE TITLE
Redirect players that voted for map change to different server with that map

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#5637]
As cheeser0613 suggested, veto could start earlier and be harder. We could only pass vote when everyone with veto pressed yes.

**Comment by fokkonaut** [comment]
I think the veto is pretty strong already.

**Comment by heinrich5991** [comment]
> We could only pass vote when everyone with veto pressed yes.

I think this is already the case.

**Comment by heinrich5991** [comment]
The problem votes solve and don't solve nicely right now is coordination of players for a new map.

Perhaps votes could redirect all yes-voting players to a new server with the given map? This would solve the coordination problem while also allowing people to continue playing a map.

**Comment by fokkonaut** [comment]
That would require a client modification, as the client cant be redirected to another server IP.

**Comment by heinrich5991** [comment]
Yup, or some advanced proxy setup. But client modification sounds better.

**Comment by fokkonaut** [comment]
That wont work for everyone then, I have talked to kamillentee about this already. I think it wouldnt be optimal, if players that voted F3 stay on the old map while new clients get moved.

**Comment by heinrich5991** [comment]
One can first ship the client change and later the server change when most people have updated already.

**Comment by Patiga** [comment]
> We could only pass vote when everyone with veto pressed yes.

I think this is the case usually, with the afk state as the exception
As long as you're afk, you don't participate in the vote and don't have your passive veto because of this

**Comment by fokkonaut** [comment]
#5662

**Comment by ChillerDragon** [comment]
#6754
